src/requirements/requirement-graph-builder-from-user-data.ts

```diff
@@ -22,16 +21,16 @@ function forfeitTransferCredit(coursesTaken: readonly CourseTaken[]): readonly C
 }
 
 export default function buildRequirementFulfillmentGraphFromUserData(
-  coursesTaken: readonly CourseTaken[]
+  coursesTaken: readonly CourseTaken[],
+  { college, major: majors, minor: minors }: AppOnboardingData,
+  toggleableRequirementChoices: AppToggleableRequirementChoices
```

Comment:
Why do we not directly use the data from `store.state`? Is this to avoid circular import?

Reply:
Yes.